rinex/src/navigation/health/gal.rs

```diff
@@ -1,88 +1,84 @@
-//! GAL Sv Health specifications
-use bitflags::bitflags;
+//! GAL SV Health specifications
 
-/// GAL Sv Health indication
-#[derive(Debug, Clone, PartialEq, PartialOrd)]
+/// GAL INAV & FNAV SV Health indication.
+/// See [Bibliography::RINEX3] for more information.
+#[derive(Default, Debug, Clone, PartialEq, PartialOrd)]
 #[cfg_attr(feature = "serde", derive(Serialize))]
-pub enum GALHealth {
-    /// GAL Legacy Sv Health indication
-    LNAV(LegacyHealth),
-    /// GAL INAV Sv Health indication
-    INAV(Health),
-    /// GAL FNAV Sv Health indication
-    FNAV(Health),
+pub struct GALHealth {
+    /// E1-B Data Validity Status
+    /// `false` = Navigation data valid
+    /// `true` = Working without guarantee
+    e1b_dvs: bool,
+    /// E1-B/C Signal Health Status
+    /// 0 = Signal OK
+    /// 1 = Signal out of service
+    /// 2 = Signal will be out of service
+    /// 3 = Signal Component currently in Test
+    e1b_hs: u8,
+    /// E5a Data Validity Status
+    /// `false` = Navigation data valid
+    /// `true` = Working without guarantee
+    e5a_dvs: bool,
+    /// E5a Signal Health Status
+    /// 0 = Signal OK
+    /// 1 = Signal out of service
+    /// 2 = Signal will be out of service
+    /// 3 = Signal Component currently in Test
+    e5a_hs: u8,
+    /// E5b Data Validity Status
+    /// `false` = Navigation data valid
+    /// `true` = Working without guarantee
+    e5b_dvs: bool,
+    /// E5b Signal Health Status
+    /// 0 = Signal OK
+    /// 1 = Signal out of service
+    /// 2 = Signal will be out of service
+    /// 3 = Signal Component currently in Test
+    e5b_hs: u8,
 }
 
-impl Default for GALHealth {
-    fn default() -> Self {
-        Self::LNAV(LegacyHealth::default())
+impl std::fmt::UpperExp for GALHealth {
+    fn fmt(&self, f: &mut std::fmt::Formatter) -> std::fmt::Result {
+        let value: u32 = u32::from(self);
+        write!(f, "{:e}", value as f32)
```

Comment:
FYI, all the NAV ecosystem implements the `UpperExp` formatting because that's how we (intend) to format those fields in file production contexts.

I say intended because NAV files production is the most problematic still to this day, while OBS and Meteo RINEX have been completed already.